numpy/core/setup.py

```diff
@@ -386,6 +386,9 @@ def check_mathlib(config_cmd):
                                "MATHLIB env variable")
     return mathlibs
 
+def check_use_legacy_division():
+    return os.environ.get('NPY_USE_LEGACY_DIVISION') is not None
```

Comment:
Was there a reason for this, I don't really see why we would want to switch to legacy division (if there was a reason, we have `doc/source/reference/global_state.rst` where we could mention it).

Reply:
I was a bit hesitant to remove the old method completely worrying about compatibility, let's keep it at compile time variables at least for one release and explore to remove it.

Reply:
If there is no reason to run tests, or because someone has legitimate reason to want to switch back, I think just delete it. Its easy to find the PR and commit that removed it to revert to the old state.

Reply:
Ohh ok yeah makes sense. Will remove it, thanks.